src/Symfony/Bundle/FrameworkBundle/Command/DebugAutowiringCommand.php

```diff
@@ -127,6 +127,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     $serviceLine .= ' - <fg=magenta>deprecated</>';
                 }
             } elseif (!$all) {
+                if ($search) {
+                    $io->text('The non-aliased services are not shown unless the all option is passed, please do not type-hint the classes if possible.');
```

Comment:
all should be quoted `"all"`.

Reply:
Stop me if I'm wrong but wording-wise I'd say you don't type-hint classes, but rather type-hint _on_ classes (or _against_ classes).

I suggest:

> Non-aliased services are not shown unless the "--all" option is passed, you would better not type-hint on these classes.